script.js

```diff
@@ -0,0 +1,100 @@
+const hamburger = document.getElementById('menuBtn');
+const quitBtn = document.querySelector('.btnClose');
+const menuPopup = document.getElementById('menu-bar');
+const menuItems = document.querySelectorAll('.menu-item');
+
+menuPopup.classList.add('hidden');
+
+function closePopup() {
+  hamburger.classList.toggle('hidden');
+  quitBtn.classList.toggle('hidden');
+  menuItems.forEach((item) => { menuPopup.remove(item); });
+  menuPopup.classList.toggle('hidden');
+}
+
+function menuDisplay() {
+  hamburger.classList.toggle('hidden');
+  quitBtn.classList.toggle('hidden');
+  menuPopup.classList.add('menuPopup');
+  menuPopup.classList.remove('hidden');
+
+  menuPopup.appendChild(quitBtn);
+  menuItems.forEach((item) => {
+    menuPopup.appendChild(item);
+    item.addEventListener('click', closePopup);
+  });
+  quitBtn.addEventListener('click', closePopup);
+}
+
+hamburger.addEventListener('click', menuDisplay);
+
+// speakers cards //
+
+const cards = document.querySelector('.container-article');
+
+const speakers = [
+  {
+    id: 'card1',
+    soloist: 'Captain Marvel',
+    photo: './images/speaker_01.png',
+    power: 'Enhanced Physical Abilities.',
+    description: "Captain Carol Danvers is a former United States Air Force pilot who, upon destroying the Lightspeed Engine made by her mentor Mar-Vell, was exposed to the Tesseract's energy, gaining cosmic powers and was abducted to serve the Kree Empire.",
+  },
+
+  {
+    id: 'card2',
+    speaker: 'Green Lantern',
+    photo: './images/speaker_02.png',
+    power: 'Power Ring',
+    description: 'Green Lantern is the alias of several fictional superheroes of the DC Universe, from the DC Comics publisher, which are characterized by carrying a ring of power and having the ability to create manifestations of solid light with said rings.',
+  },
+
+  {
+    cardid: 'card3',
+    speaker: 'Doctor Strange',
+    photo: './images/speaker_03..png',
```

Comment:
- [ ] Please make sure to check your images, as this one is not being displayed on the screen. Maybe because of an issue with the path or because of any problem accessing the object.